docs/issues/content-separation.md

```
@@ -0,0 +1,391 @@
+- [x] **AC-11**: ローカルで `pnpm dev` を実行すると、全ページ・タグページが表示される
+- [x] **AC-12**: 非公開コンテンツの取得に失敗しても、ローカルの既存ビルドツリーが消されない（オフライン時に直前の状態が保たれる）
+- [x] **AC-13**: コンテンツを増減しても公開リポジトリ側の設定変更が不要である（`.gitignore` などの追記が発生しない）
+- [x] **AC-14**: `pnpm dev` 実行中に作業クローンの Markdown を保存すると、サイトがホットリロードされる
+
+---
+
+## 設計
+
+### 全体構成
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)